pkg/cmd/dcos_cluster_list.go

```diff
@@ -0,0 +1,122 @@
+package cmd
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"sync"
+
+	"github.com/dcos/dcos-cli/pkg/cli"
+	"github.com/spf13/cobra"
+)
+
+// newCmdClusterList lists the clusters.
+func newCmdClusterList(ctx *cli.Context) *cobra.Command {
+	var onlyAttached bool
+	var jsonOutput bool
+	cmd := &cobra.Command{
+		Use:  "list",
+		Args: cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clusters, err := ctx.Clusters()
+			if err != nil {
+				return err
+			}
+
+			currentCluster, err := ctx.Cluster()
+			if err != nil {
+				return err
```

Comment:
This shouldn't stop the command's execution with an error, it is fine to continue with an empty currentClusterID here.